input/pagecontent/StructureDefinition-au-erequesting-servicerequest-imag-intro.md

```diff
@@ -1,11 +1,27 @@
 {% include rrv-dev-note.md %}
 
 ### Profile specific implementation guidance
-- `ServiceRequest.code`:
-  - **SHOULD** be selected from the preferred <a href="ValueSet-ranzcr-radiology-referral.html">RANZCR Radiology Referral</a> value set
-  - if a system is unable to provide a code from the preferred <a href="ValueSet-ranzcr-radiology-referral.html">RANZCR Radiology Referral</a> value set, it is preferred to select a suitable code from the SNOMED CT <a href="https://healthterminologies.gov.au/fhir/ValueSet/imaging-procedure-1">Imaging Procedure</a> value set over only supplying local codes
-  - coding is preferred over supplying only free text with no codes
-- The use of coding can vary significantly across systems, implementers need to understand that they may encounter codes they do not recognise and be prepared to handle those resources appropriately. Systems **SHALL** populate `ServiceRequest.code.text` so that systems can at least display the service requested even if the system does not recognise the code supplied.
-- See guidance on the construction of identifiers in each Identifier profile and the section on [Business Identifiers](https://build.fhir.org/ig/hl7au/au-fhir-base/generalguidance.html#business-identifiers) in AU Base.
+<br/>
+**ServiceRequest.code**
```

Comment:
Commenting here to get an inline comment. Apologies for missing this earlier, but now that the subheadings have been removed from ServiceRequest.code, there is inconsistency in how implementation guidance is presented. The bullet point about the provision of insurance feels lost in the content and it is unclear where it belongs. 
I suggest adding a bullet point to ServiceRequest.code for clarity and keeping the resolution guidance content under that bullet point.
@michael-wilson-au , thoughts? This could also be a backlog item to be addressed later.

Reply:
Yes agreed. Can this be done as part of this PR? It is just adjusting the formatting to ensure the new content displays correctly?

Reply:
@JayMurdoch, could you please fix in both intro.md files? thanks

Reply:
Amended in both files, thanks.

Reply:
Please remove bold from ServiceRequest.code in both instances.

Reply:
Amended, thanks.